### ISSUE TITLE
Add accessible behaviour to Topic pages

### ISSUE THREAD

**Issue by nicola-d-bbc** [bbc/simorgh#9970]
There are three accessibility requirements missing from the new Topic pages:

- The 'Skip to content' link should anchor to the h1 when used
- Add specific colour to h1 element as specified in the design (e.g. do not inherit a colour)
- Focus state of promo links should include an underline as well as the browser default outline (currently it uses browser default only)
- Add visited link styling to links in promos

**To Reproduce**
*Skip to content h1*
Steps to reproduce the behaviour:

1. Go to 'https://www.test.bbc.com/pidgin/new_topics/cqywjyzk2vyt'
2. Tab to the 'Skip to content' link and press enter
3. Page stays at top 

**Expected behaviour**
Page should move down to anchor where the h1 begins (example can be seen here: https://www.bbc.com/pidgin)

*Specific colour to h1 element* (Firefox example)
Steps to reproduce the behaviour:
1. Go to 'https://www.test.bbc.com/pidgin/new_topics/cqywjyzk2vyt'
2. Change colours to custom colours and enable
3. Disable custom colours
4. H1 element is still custom colour

**Expected behaviour**
H1 text colour should change back to default colour

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
